modules/rtbdemandLiteBidAdapter.js

```diff
@@ -0,0 +1,360 @@
+/* eslint dot-notation:0, quote-props:0 */
+import {logError, getTopWindowLocation} from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_SERVER = 'bidding.rtbdemand.com';
+
+const NATIVE_DEFAULTS = {
+  TITLE_LEN: 100,
+  DESCR_LEN: 200,
+  SPONSORED_BY_LEN: 50,
+  IMG_MIN: 150,
+  ICON_MIN: 50,
+};
+
+/**
+ * Rtbdemand "Lite" Adapter.  This adapter implementation is lighter than the
+ * alternative/original RtbdemandAdapter because it has no external
+ * dependencies and relies on a single OpenRTB request to the Rtbdemand
+ * bidder instead of separate requests per slot.
+ */
+export const spec = {
+
+  code: 'rtbdemandLite',
+
+  aliases: ['rtbdemandLite'],
```

Comment:
It's unnecessary to define an alias that's the same as the code.  This can be removed if you don't need an alias.